volatility3/framework/symbols/windows/mft.json

```diff
@@ -192,7 +192,7 @@
                         "name": "unsigned int"
                     }
                 },
-                "AlocatedSize": {
```

Comment:
This one might cause problems if people already rely on it.  I'd need to have a think about changing it.  Probably it's ok, but I need to consider what it might break.

Reply:
This looks to be the only place it's used, so I'll take the risk and change it.